internal/config/config.go

```diff
@@ -314,6 +315,156 @@ func setClowderConfiguration(v *viper.Viper, clowderConfig *clowder.AppConfig) {
 	// Metrics configuration
 	v.Set("metrics.path", clowderConfig.MetricsPath)
 	v.Set("metrics.port", clowderConfig.MetricsPort)
+
+	// Check TlsCAPath if present
+	if clowderConfig.TlsCAPath != nil {
+		envSSLCertDir := os.Getenv("SSL_CERT_DIR")
+		if !checkPathInList(*clowderConfig.TlsCAPath, envSSLCertDir) {
+			slog.Error("clowder provides a path for CA, but the path is not found at SSL_CERT_DIR",
+				slog.String("TlsCAPath", *clowderConfig.TlsCAPath),
+				slog.String("SSL_CERT_DIR", envSSLCertDir),
+			)
+			panic("SSL_CERT_DIR missconfiguration")
+		}
+	}
```

Comment:
OK, thinking aloud even further... if we have TlsCAPath configured, but it is not in `SSL_CERT_DIR` - couldn't we add it to the env var?  (Just seeking comment - I don't want to block this PR on such a change, but maybe it is a way to simplify the configuration in the future).

Reply:
good question! I think I can get it working! I am not a fun of `init` functions, but I think it will make this work in that way; I mean, as this depends on `clowder` package, the clowder configuration is already loaded, I can check the values, and update the environment variable. Thanks! :+1:

Reply:
@frasertweedale , I have updated and checked! and seems to work; I checked in a container launched locally, using a custom clowder configuration (just to check the cert pools); the SSL_CERT_DIR was updated, and I added a self-signed certificate which was loaded (just checked he number of certificates differed from one execution and other); be aware that `SystemCertPool` use in the process (deeper calls) a `once` semantic so the pool of system certificates are loaded only once during the execution. More details on later comments.

Reply:
OK, very nice!